stream-compositions/services/transaction-composition-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,59 @@
+spring:
+  zipkin:
+    enabled: false
+  security:
+    oauth2:
+      client:
+        registration:
+          dbs:
+            authorization-grant-type: client_credentials
+            client-id: bb-client
+            client-secret: bb-secret
+            client-authentication-method: post
+        provider:
+          dbs:
+            token-uri: http://token-converter:8080/oauth/token
+  main:
+    allow-bean-definition-overriding: true
+  codec:
+    max-in-memory-size: 10MB
+sso:
+  jwt:
+    internal:
+      type: signed
+      signature:
+        algorithm: HS256
+        key:
+          type: ENV
+          value: SIG_SECRET_KEY
+backbase:
+  mtls:
+    enabled: false
+  security:
+    http:
+      enabled: true
+    service-api:
+      authentication:
+        required-scope: api:service
+  stream:
+    legalentity:
+      sink:
+        useIdentityIntegration: true
+    dbs:
+      user-manager-base-url: http://user-manager-nextgen:8080
+      access-control-base-url: http://access-control-nextgen:8080
+      arrangement-manager-base-url: http://arrangement-manager-nextgen:8080
+      transaction-manager-base-url: http://transaction-manager-nextgen:8080
+    identity:
+      identity-integration-base-url: http://identity-integration-service:8080
+
+    compositions:
+      transaction:
+        enable-completed-events: false
+        enable-failed-events: true
+        transaction-integration-url:  http://wsecu-transaction-integration-service-nextgen:8080
```

Comment:
maybe we could keep these project-specific properties in a separate spring profile.